doc/source/force-constants.rst

```diff
@@ -24,14 +24,12 @@ The :py:attr:`ForceConstants.force_constants <euphonic.force_constants.ForceCons
   \frac{\partial^{2}E}{{\partial}u_{\kappa\alpha0}{\partial}u_{{\kappa}'{\alpha}'l}}
 
 This describes the change in total energy when atom :math:`\kappa` in unit cell :math:`0`` is displaced in direction :math:`\alpha` and atom :math:`\kappa\prime` in unit cell :math:`l` is displaced in direction :math:`\alpha\prime`.
-In Euphonic the force constants are stored in 'compact' form, which means that the minimum required information is stored.
-Using equivalent vectors, the force constants between every atom in the unit cell, and every atom in the supercell are all that is needed,
-so only :math:`N(3n)^2` values need to be stored rather than :math:`(3Nn)^2` as in the 'full' square force constants matrix,
-where N is the number of unit cells in the supercell, and n is the number of atoms in the unit cell.
+In Euphonic the force constants are stored in 'compact' form, with shape :math:`3Nn\times3n`, where :math:`N` is the number of cells in the supercell, and :math:`n` is the number of atoms in the unit cell.
```

Comment:
This description is contradicted by the following paragraph.

Reply:
You mean about the shape of the matrix?

Reply:
Right. Maybe instead of "with shape 3Nnx3n" it should be something like "with N(3n^2) elements"? Then we build anticipation for the next paragraph telling you the actual shape!

Reply:
Good idea, done!